src/Stats/Views/Stats/StatsHtmlView.php

```diff
@@ -74,7 +74,7 @@ public function render()
 
 		$data['total'] = count($items);
 
-		$this->renderer->set('data', $data);
+		$this->getRenderer()->set('data', $data);
```

Comment:
`$this->addData('data', $data);` - At least use the view API instead of directly injecting into the renderer